isis/tests/BundleSettingsTests.cpp

```diff
@@ -0,0 +1,17 @@
+#include "BundleSettings.h"
+
+#include <gtest/gtest.h>
+
+class BoolTest : public ::testing::TestWithParam<bool> {
+  // Intentionally empty
+};
+
+TEST_P(BoolTest, validateNetwork) {
+  BundleSettings testSettings;
+  testSettings.setValidateNetwork(GetParam());
+  EXPECT_EQ(GetParam(), testSettings.validateNetwork());
+}
+
+INSTANTIATE_TEST_CASE_P(BundleSettings,
+                        BoolTest,
+                        ::testing::Bool());
```

Comment:
`::testing::Bool()` produces `{true, false}`
Another way to write this would be ::testing::Values(true, false)

Reply:
There's also a few other handy options: https://github.com/google/googletest/blob/master/googletest/docs/advanced.md#value-parameterized-tests